packages/commonwealth/client/scripts/views/modals/AuthModal/useAuthentication.tsx

```diff
@@ -500,7 +500,7 @@ const useAuthentication = (props: UseAuthenticationProps) => {
             })
           : selectedAddress,
       ],
-      profileChainIds: [app.activeChainId() ?? wallet.chain],
```

Comment:
the intention of removing is to find any matching address that already exists regardless of the chain?

Reply:
Yes

Reply:
Sounds good, thanks!